doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc

```diff
@@ -1,24 +1,95 @@
 [[my-services-tab]]
 
-== The My Services Tab
+== Services Tab
 
-The *My Services* tab lists all available services with a summary of information associated with each service, including *Service Name*, *Retirement Date*, *Number of VMs*, *Owner*, the date it was *Created On*, and its *Power State*. You can filter or sort services by those entities, and now also by *Relative Cost*. There is now the ability for users to perform power operations on virtual machines within the Self Service user interface using the new set of buttons, *Start*, *Stop*, and *Suspend* for each service.  
+The *Services* tab lists all available services with a summary of information associated with each service, including the *Service Name*, its current *Power State* represented by an icon, *Number of VMs*, *Owner*, date *Created*, and *Retires* date. Additionally, you can see the chargeback cost per service for a virtual machine's month-to-date usage. You can filter or sort services by those entities, and by *Relative Cost*. 
 
-image:SSUI_MyServices_tab.png[]
+On the same screen, you have the option to edit the service name and description, edit tags, set ownership, retire the service now or at a later date, or remove the service. You can also perform power operations on virtual machines by clicking more options _(represented by three dots)_ next to the service.
 
-Click a service to view further details including information on the virtual machines associated with it. There is now ability for users to view chargeback data per service for the virtual machines' month-to-date usage. You can also get HTML 5 console access to the virtual machine by clicking the image:SSUI_html5_icon.png[Html 5 Icon] button, as highlighted on the screen capture. On the same screen, you also have options to retire the service now or at a later date, edit or remove the service. Additionally, you can now change the ownership and group of the service using the *Set Ownership* button, and also *Reconfigure* a service, similar to in the full administrative user interface. 
+image:ssui-services-menu.png[]
 
-image:SSUI_Service_Details1.png[]
+There is now the option to perform most of these tasks in bulk (except editing service details and power operations). In order to do so, select the services and then click the designated button for the required task on the upper-left corner of the screen. 
```

Comment:
[WIP] Edited "There is now the option to perform "
Note - I have changed "left" to "right" as I'm pretty sure this was the intended wording. But if you can double check that would be great. :)